dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.jaxwscdi-2.3.feature

```diff
@@ -0,0 +1,13 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.jaxwscdi-2.3
```

Comment:
Only a minor point but this naming kinda suggests it applies to cdi-2.3 (which doesn't exist yet). Why not name it `jaxws2.3-cdi`? Same goes for the bundle naming `com.ibm.ws.jaxws.2.3.cdi`

Reply:
I'll let you guys decide that as a squad ;-)